packages/docs/pages/header.tsx

```diff
@@ -85,24 +86,34 @@ const HeaderPage = () => {
           id="props"
           routes={[
             {
-              id: 'header-props',
+              id: 'header',
               title: 'Header',
               render: () => <HeaderPropTable />,
             },
             {
-              id: 'action-button-props',
+              id: 'action-button',
               title: 'ActionButton',
-              render: () => <ActionButtonPropsTable />,
+              render: () => (
+                <ActionButtonPropsTable
+                  id="action-button-prop-table"
+                  inheritedProps={<ButtonPropTable collapsible />}
```

Comment:
We like to use `inheritedProps` to showcase they are extended. The only issue is that we can't omit things from these tables. How we do props in the documentation needs to be revamped which can be for the future.